fe/fe-core/src/main/java/org/apache/doris/nereids/rules/analysis/BindExpression.java

```diff
@@ -564,10 +567,16 @@ protected boolean condition(Rule rule, Plan plan) {
             ),
             RuleType.BINDING_RESULT_SINK.build(
                 unboundResultSink().then(sink -> {
-                    List<NamedExpression> outputExprs = sink.child().getOutput().stream()
-                            .map(NamedExpression.class::cast)
-                            .collect(ImmutableList.toImmutableList());
-                    return new LogicalResultSink<>(outputExprs, sink.child());
+
+                    final ImmutableListMultimap.Builder<ExprId, Integer> exprIdToIndexMapBuilder =
```

Comment:
not see how to process ctas

Reply:
Ctas stmt is only supported by legacy, It's already supported in another  pr [Infer the column name when create view if the column is expression](https://github.com/apache/doris/pull/24990), and when cats is supported in nereids future, this is also useful.